apps/api/src/app/workflows-v2/workflow.controller.ts

```diff
@@ -208,24 +223,8 @@ export class WorkflowController {
     );
   }
 
-  @Patch('/:workflowId/steps/:stepId')
```

Comment:
👋 The PATCH step endpoint wasn't used in the Dashboard anymore. Most importantly, it's a problematic endpoint that couldn't survive our modelling.

Steps depend on each other through variables. So, PATCHing a single step without considering the other steps shouldn't be allowed. Imagine patching a digest step that affects all the other steps that follow.